### DISCUSSION TITLE
collapse lists

### DISCUSSION THREAD

**Discussion by dev-hanzo** [ObsidianPublisher/obsidian-github-publisher#71] [Ideas]
take a look if you interested. github ability the collapse lists!
https://github.com/dkorolev/collapsible_markdown
**it would be great use it is in plugin**

![image](https://user-images.githubusercontent.com/117753674/211795326-0f0ecbc3-f451-46a6-aa24-6062861cb82d.png)

**Comment by Mara-Li** [comment]
You can make markdown working in HTML using the extension: https://python-markdown.github.io/extensions/md_in_html/

See here to use it in Material Mkdocs : https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html
